arrow-core/src/main/kotlin/arrow/core/extensions/set/foldable/SetKFoldable.kt

```diff
@@ -30,15 +30,15 @@ import kotlin.jvm.JvmName
 @Deprecated(
```

Comment:
The deprecations in this file not correct. See the `List` deprecation for examples.